templates/README.md

```diff
@@ -44,6 +44,9 @@ Your repository comes with a end-to-end CI/CD pipeline, which includes the follo
 Your application is assumed[(ref)][base-deployment-secret] to rely on a database(RDS), In your Kubernetes
 application namespace, an application specific user has been created for you and hooked up to the application already.
 
+## Cache service
```

Comment:
I don't think this is useful, let's take it out. Also it may not be accurate if they choose not to use a cache.